homeworks/vasyl.biliak_VasylBiliak/GamesProject/games.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="./style/games.css">
+        <title>Games</title>
+    </head>
+    <body>
+        <header class="header">
+            <div class="header__box">
+                <img class="header__logo" src="img/logo.png" alt="logo" width="90px" height="90px">
+                <div class="header__links">
+                    <a class="header__link" href="./games.html">Games</a>
+                    <a class="header__link" href="./index.html">About</a>
+                </div>
+            </div>
+        </header>
+        <main>
+            <div class="contain">
+                <div class="content">
+                    <h1 class="heading"> Study project "Games"</h1>
+                </div>
+                <div class="game__cards">
+                    <div class="game__card">
+                        <div class="card__content">
+                            <div class="card__content__img">
+                                <img src="./img/img_3.png" alt="" width="90px" height="90px">

```

Comment:
...and add to the image class name of BEM-element: `game-card__img`. After you can add styles to image by this class, not by tag `img` after cascade, like you do [here](https://github.com/MastersAcademy/fe-course-basic-2023/pull/136/files#diff-67d42e489b60c95322d63dc1064485e8287a4c5a1548f7554168f67f57c3b30cR106)